src/components/GlobalPreferences/CustomLabels/LocalIdentities.js

```diff
@@ -340,7 +340,7 @@ function Actions({ onExport, onRemove, onShare, disabled }) {
             css={`
               height: 24px;
               $textStyle('body2');
-              color: ${theme.surfaceContent};
+              color: ${disabled ? theme.hint : theme.surfaceContent};
```

Comment:
Just as a note, we should probably have a `theme.surfaceContentDisabled` (`hint` is meant to be used for `placeholder` and things like that). /cc @dizzypaty

Reply:
Sure. Can we use hint for this or do the whole add new color round trip?

Reply:
Yes let’s use `hint` for now, until we have this new color!